packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -165,31 +137,6 @@ describe('TransactionService', function() {
     tr2.detectFinish()
   })
 
-  it('should sendPageLoadMetrics', function(done) {
-    config.set('active', true)
-    config.set('capturePageLoad', true)
-
-    transactionService = new TransactionService(logger, config)
-
-    transactionService.subscribe(function() {
-      expect(tr.isHardNavigation).toBe(true)
-      expect(tr.marks.agent).toBeDefined()
-      expect(tr.marks.navigationTiming).toBeDefined()
-      done()
-    })
-    var tr = transactionService.sendPageLoadMetrics('test')
-
-    transactionService = new TransactionService(logger, config)
-    var zoneTr = new Transaction('test-name', 'test-type', {
-      canReuse: true
```

Comment:
I think we should still keep this test because of the `canReuse` but maybe refactor it to use startTransaction instead of `sendPageLoadMetrics`

Reply:
Done.